packages/babel-helpers/src/helpers.js

```diff
@@ -399,7 +399,7 @@ helpers.objectSpread = helper("7.0.0-beta.0")`
       var source = (arguments[i] != null) ? arguments[i] : {};
```

Comment:
I feel like we can add `Object` to `arguments[i]` and forget about `Object(source)` later.

Reply:
you mean like this?
```
var source = (arguments[i] != null) ?  Object(arguments[i]) : {};
```

Reply:
fixed https://github.com/babel/babel/pull/10902/commits/1366ed3c1ef80dbe4b11996b0da8de08eceb4ac8